dodiscover/replearning/gin.py

```diff
@@ -0,0 +1,113 @@
+"""
+Wrapper for the GIN algorithm in causal-learn
+TODO: Need type hints
+"""
+from causallearn.search.HiddenCausal.GIN.GIN import GIN as GIN_
+from pywhy_graphs import CPDAG
+
+class GIN:
+    """Wrapper for GIN in the causal-learn package.
+
+    Parameters
+    ----------
+    indep_test_method : str
+        The method to use for testing independence, by default "kci"
```

Comment:
I would add a link to causal-learn's method documentation, so users know what other options they can pass in.

Reply:
[e9fb4ca](https://github.com/py-why/dodiscover/pull/72/commits/e9fb4ca2c41b0d5aad2020c1afdedc796d352a2c)